packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -222,6 +229,120 @@ describe('Sayt Component', () => {
     });
   });
 
+  describe('requestSayt()', () => {
+    let dispatchEvent, query, searchbox;
+    beforeEach(() => {
+      query = 'some-query';
+      searchbox = 'some-searchbox-id';
```

Comment:
Since this doesn't change in the tests, this should just be a `const` outside the `beforeEach`.